wagtailautocomplete/views.py

```diff
@@ -1,7 +1,8 @@
+from urllib.parse import unquote
```

Comment:
Can you find a Python 2.7 compat fallback here? Python 2.7 support is still desirable for another couple months (until Wagtail 1.13 goes away for real in April).